### DISCUSSION TITLE
关于评论样式的问题！

### DISCUSSION THREAD

**Discussion by Yuumi0221** [YunYouJun/hexo-theme-starry#172] [Q&A 疑难杂症] [answered]
## [-Yuumi’s Blog-](https://yuumi0221.github.io/)

**blog: Yuumi的烟波之路**

**url: https://yuumi0221.github.io/**

**avatar: https://cdn.jsdelivr.net/gh/Yuumi0221/Yuumi0221.github.io/images/settings/favicon.png**

**desc: いつか伝えに行くよ**

------

真的非常喜欢云游君的主题🥰！没想到就连我这样的前端小白，也可以跟着使用说明一步步完成我的主页ww，做完真的成就感满满！！（虽然具体的内容还没有加——）

同时我又有一个小问题想问一下大佬了😵按文档要求添加了Waline的评论功能之后，在夜间模式时评论区还是白白一片……

在waline文档中有写到：

> 使用 `@media` 选择器通过 `prefers-color-scheme` 来根据设备颜色模式状态自动切换，你只需将 `dark` 设置为 `'auto'`。

但是配置完好像并没有成功。。看了云游君的代码之后还是没有找到相应能改的代码orzzz，还是想拜托云游君能指点一下，谢谢！！

**Comment by YunYouJun** [comment, marked as answer]
谢谢（自定义的很好看 hhh）

如果你使用 Waline 的话，亮暗模式这个不用操心，不需要额外的配置，主题已经默认支持啦！（不过在 dev 预览版分支，你可能需要切换下分支）

**Reply by Yuumi0221** [reply]
欧原来在新版中！！我这就试试看！谢谢你www
